python/semantic_kernel/connectors/telemetry.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import os
+
+import pkg_resources
+
+TELEMETRY_DISABLED_ENV_VAR = "AZURE_TELEMETRY_DISABLED"
+
+IS_TELEMETRY_ENABLED = (
+    os.environ.get(TELEMETRY_DISABLED_ENV_VAR, "false").lower() != "true"
+)
+
+HTTP_USER_AGENT = "Semantic-Kernel"
+
+APP_INFO = {
+    "name": HTTP_USER_AGENT if IS_TELEMETRY_ENABLED else "",
+    "version": pkg_resources.get_distribution("semantic-kernel").version
+    if IS_TELEMETRY_ENABLED
+    else "",
+    "url": "",
```

Comment:
nit: Is `url` property required if it's empty?

Reply:
yes, I hit a lot of issues until I filled out the field. 

https://github.com/openai/openai-python/blob/284c1799070c723c6a553337134148a7ab088dd8/openai/api_requestor.py#L153

This line assumes that "url" exists and will raise an exception if it does not.

Reply:
Makes sense, thank you!